cmd/compute-domain-controller/computedomain.go

```diff
@@ -33,8 +33,15 @@ import (
 type GetComputeDomainFunc func(uid string) (*nvapi.ComputeDomain, error)
 
 const (
+	// informerResyncPeriod defines how often the informer will resync its cache
+	// with the API server. This helps ensure eventual consistency.
 	informerResyncPeriod = 10 * time.Minute
 
+	// mutationCacheTTL defines how long mutation cache entries remain valid.
+	// This should be long enough for the informer cache to catch up but
+	// not so long that stale entries cause issues.
+	mutationCacheTTL = time.Hour
```

Comment:
Thanks. Most importantly, we have a bit of discussion around these time constants now on the PR, and don't need to reflect everything in a code comment.